cpg_workflows/inputs.py

```diff
@@ -277,11 +295,13 @@ def _populate_analysis(cohort: Cohort) -> None:
             dataset.get_sequencing_group_ids(),
             analysis_type=AnalysisType.GVCF,
             dataset=dataset.name,
+            meta=get_gvcf_meta(),
```

Comment:
nitpick, but these get_X_meta functions are called a bunch of times in this method, might be better to store the return value as a variable.

Reply:
classic me behaviour, will fix